src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import { useTodos } from '../../Store';
+import { getFilteredTodos } from '../../services/getFilteredTodos';
+import { TodoItem } from '../TodoItem';
+
+export const Todolist = () => {
+  const { todos, filterStatus } = useTodos();
+
+  const filteredTodos = getFilteredTodos(todos, filterStatus);
```

Comment:
this can be memoized as well

Reply:
Still not fixed 
<img width="858" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/64750958/5fcf60f3-0cef-4129-9fc2-3bc26dbcce6f">